src/component/Registration/Signup.css

```diff
@@ -0,0 +1,98 @@
+.form {
+    background-color: #fff;
+    color: black;
+    width: 400px;
+    border-radius: 8px;
+    padding: 8px 8px; 
+  }
+
+  section{
+      height: 100vh;
+      width: 100%;
+      background-color: aliceblue;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+  }
+
+  .container{
+      width: 90%;
+      max-width: 500px;
+      margin: 0 auto;
+      padding: 20px;
+      box-shadow: 0px 0px 20px #00000010;
+      border-radius: 8px;
+      margin-bottom: 20px;
+  }
+
+  .form-group {
+      width: 100%;
+      margin-top: 20px;
+      font-size: 20px;
+  }
+
+  .form-group input,
+  .form-group textarea {
+      width: 100%;
+      padding: 5px;
+      font-size: 18px;
+      border: 1px solid #5ea51d;
+  }
+
+  .SigninsubmitBtn{
+      width: 74%;
```

Comment:
Try to be consistent with the className naming

Reply:
done also